pkg/maps/nat/nat.go

```diff
@@ -240,18 +240,13 @@ func (m *Map) DeleteMapping(key tuple.TupleKey) error {
 	return deleteMapping6(m, key.(*tuple.TupleKey6Global))
 }
 
-func maps(ipv4, ipv6 bool) []*Map {
-	result := make([]*Map, 0, mapCount)
+// GlobalMaps returns all global NAT maps.
+func GlobalMaps(ipv4, ipv6 bool) (ipv4Map, ipv6Map *Map) {
```

Comment:
Looks good, I presume this works fine with the way we handle commands: https://github.com/cilium/cilium/blob/319ccde9f19deaa435dd6d29a84ee13cfcea90fa/cilium/cmd/bpf_nat_flush.go#L42

Reply:
Ok, judging from travis, looks like this needs to be fixed for the cli.

Reply:
Yep, fixed in latest force push